packages/graphql/src/graphql/directives/relationship.ts

```diff
@@ -17,7 +17,14 @@
  * limitations under the License.
  */
 
-import { DirectiveLocation, GraphQLBoolean, GraphQLDirective, GraphQLList, GraphQLNonNull, GraphQLString } from "graphql";
+import {
```

Comment:
I suppose this happened when saving this file at some point. But we shouldn't include it as part of this PR

Reply:
whoops!

Reply:
https://github.com/neo4j/graphql/pull/3468/commits/06ee128ff8fd241ca1c9a2d55f2e8b0283a02c08